web/source/kmwembedded.ts

```diff
@@ -124,10 +124,10 @@ namespace com.keyman.osk {
 }
 
 namespace com.keyman.text {
-  let nativeDefaultKeyOutput = Processor.prototype.defaultKeyOutput;
+  let nativeForBaseKeys = DefaultOutput.forBaseKeys;
 
   // Overrides the 'native'-mode implementation with in-app friendly defaults prioritized over 'native' defaults.
-  Processor.prototype.defaultKeyOutput = function(this: Processor, Lkc: KeyEvent, keyShiftState: number, usingOSK: boolean): string {
+  DefaultOutput.forBaseKeys = function(this: Processor, Lkc: KeyEvent, keyShiftState: number): string {
```

Comment:
The same style of override seen here for use in "embedded-mode only" can be used to allow the `applyDOMCommand()` function to "extend" the headless `applyCommand` from a DOM-aware module once we're ready to fully go headless.

Reply:
... should _probably_ rework the `this` reference though - it's a static method now!